examples/plotting/lasco_overlay.py

```diff
@@ -0,0 +1,61 @@
+"""
+==============================================
+Overlay an AIA image on a LASCO C2 coronagraph
+==============================================
+
+This example shows the steps needed to overlay the disc and off-limb
+components of an AIA image within the masked occulter of a LASCO C2 image.
+
+It also demonstrates the usage of ``hypy`` to download data from the Helioviewer Project.
```

Comment:
```suggestion
```
This is not the key function of the example, so for clarity it would be better to remove this.